docs/application-connector/08-11-get-api-specification.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Get the API specification for AC components
+type: Tutorials
+---
+
+You can get the API specification directly from this website. When you are in the **Application Connector** component, navigate to the **API Consoles** section on the right and choose the desired API to browse its specification, see it on Github, or download it.
```

Comment:
I don't think it's a good idea to provide such a lengthy description of where to find the element on the website. A link and a short comment would be enough IMO.